mungegithub/github/github.go

```diff
@@ -1851,13 +1860,30 @@ func (config *Config) ForEachIssueDo(fn MungeFunction) error {
 			if err != nil {
 				return err
 			}
+			if obj.Issue.Number != nil {
+				delete(extraIssues, *obj.Issue.Number)
+			}
 		}
 		if response.LastPage == 0 || response.LastPage <= page {
 			break
 		}
 		page++
 	}
 	config.since = since
+
+	// Handle additional issues
+	for id := range extraIssues {
```

Comment:
It'd be pretty awesome if we could do these async instead of waiting for the next loop, but this is certainly safest. I wonder how many of the mungers can handle getting called twice at the same time...